src/src/rx-serial/SerialMavlink.cpp

```diff
@@ -156,7 +156,6 @@ void SerialMavlink::sendQueuedData(uint32_t maxBytesToSend)
             }
             else
             {
-                uint8_t buf[MAVLINK_MAX_PAYLOAD_LEN];
```

Comment:
This one would have mostly been fine because very few messages use up the full payload length.